docs/docs/sample-scripts/entra/find-obsolete-m365-groups/index.mdx

```diff
@@ -0,0 +1,288 @@
+      $groupInfo.TestStatus = "🟢 OK"
+
+      Write-Host "☀︎ $($groupInfo.Reference.displayName)"
+
+      # Tests
+      Test-GroupMembership -Group $groupInfo
+      Test-SharePointActivity -Group $groupInfo
+      Test-ConversationActivity -Group $groupInfo
+
+      # Report
+      New-Report -Group $groupInfo
+    }
+    
+    #Give feedback to user
+    Write-Host "`n-------------------------------------------------------------------"
+    Write-Host "`SUMMARY" -ForegroundColor DarkGreen
+    Write-Host "`-------------------------------------------------------------------"
+    Write-Host "`n👉 Found $($Global:ObsoleteGroups.Count) group$($Global:ObsoleteGroups.Count -gt 1 ? 's' : '') with possibly low activity."
+    Write-Host "`   Please review the report: " -NoNewline
+    Write-Host "$($Script:ReportPath)" -ForegroundColor DarkBlue
+  }
+
+  function Test-GroupMembership {
+    [CmdletBinding()]
+    param (
+      [Parameter(Mandatory = $true)] [GroupInfo] $Group
+    )
+
+    $users = m365 entra m365group user list --groupId $Group.Reference.id | ConvertFrom-Json
+    $owners = $users | Where-Object { $_.userType -eq "Owner" }
+    $members = $users | Where-Object { $_.userType -eq "Member" }
+    $guests = $users | Where-Object { $_.userType -eq "Guest" }
+
+    $Group.Membership = [ordered] @{
+      Owners  = $owners
+      Members = $members
+      Guests  = $guests
+    }
+
+    if ($owners.Count -eq 1 -and ($members.Count + $guests.Count) -eq 0) {
+      Write-Host "  → potentially obsolete (abandoned group: only 1 owner left)" -ForegroundColor Yellow
+      $reason = "Low user count"
+      
+      $Group.Membership.Status = "Abandoned ($reason)"
+      $Group.TestStatus = "🟡 Warning"
+      $Group.Reasons += $reason
+      
+      try {
+        $Global:ObsoleteGroups.Add($Group.Reference.id, $Group)
+      }
+      catch {}
+    }
+  }
+
+  function Test-SharePointActivity {
+    [CmdletBinding()]
+    param (
+      [Parameter(Mandatory = $true)] [GroupInfo] $Group
+    )
+
+    function Get-ParsedDate {
+      [CmdletBinding()]
+      param (
+        [Parameter(Mandatory = $true)] [String] $JavascriptDateString
+      )
+
+      $dateParts = [regex]::Matches($JavascriptDateString, '\d+') | ForEach-Object { $_.Value }
+
+      # Convert the parts to integers
+      $year = [int]$dateParts[0]
+      $month = [int]$dateParts[1] + 1
+      $day = [int]$dateParts[2]
+      $hour = [int]$dateParts[3]
+      $minute = [int]$dateParts[4]
+      $second = [int]$dateParts[5]
+      # $millisecond = [int]$dateParts[6]
+
+      # return a DateTime object
+      $dateObject = New-Object -TypeName DateTime -ArgumentList $year, $month, $day, $hour, $minute, $second
+      $dateObject
+    }
+    
+    $WarningDate = (Get-Date).AddDays(-90)
+    
+    # Not possible to retrieve property 'LastContentModifiedDate' via command 'm365 spo site get --url $group.siteUrl', so we need to use filtering:
```

Comment:
Shouldn't this comment be placed a few lines lower?

Reply:
Sorry for this confiusion, I've had to remove the following line that was commented out.